pkg/injector/pod_patch.go

```diff
@@ -274,7 +294,12 @@ func getMetricsPort(annotations map[string]string) int {
 }
 
 func getAppID(pod corev1.Pod) string {
-	return getStringAnnotationOrDefault(pod.Annotations, appIDKey, pod.GetName())
+	id := getStringAnnotationOrDefault(pod.Annotations, appIDKey, "")
+	if id != "" {
+		return id
+	}
+
```

Comment:
Please add TODO comment for all deprecated annotations. This way helps us to recognize it later.

Reply:
Have added a comment [at the top](https://github.com/dapr/dapr/pull/1865/files#diff-7fadd8b154bfb3e50073e295c476aa6eR80) already, guess that should be sufficient?

Reply:
Ah ok